src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.cpp

```diff
@@ -327,7 +371,15 @@ void WebRTCTransportProviderServer::HandleProvideOffer(HandlerContext & ctx, con
         // If both video and audio stream IDs are missing, return CONSTRAINT_ERROR.
         if (!videoStreamID.HasValue() && !audioStreamID.HasValue())
         {
-            ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::ConstraintError);
+            ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Protocols::InteractionModel::Status::ConstraintError);
```

Comment:
Again, this needs tracking to fix to the right status once https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11333 lands.... and do it independently of whether webRTCSessionID is null.